src/MakeItEasy/Syntax/Maker.cs

```diff
@@ -24,7 +24,7 @@ public sealed class Maker<T>
         /// System Under Test for some test.
         /// </summary>
         /// <returns>The desired object.</returns>
-        public T From()
+        public T FromDefaults()

```

Comment:
@thomaslevesque, you said

> I think my favorite options are `.FromNothing()` (for consistency with From) and `.WithDefaults()` (because it best describes what it does). But if you decide to go with .Please(), that's fine too!

I misremembered your favourites while I was off thinking about this, and ended up blending them. Then the more I thought, the more I liked the result, but I'm keen to hear your thoughts!

Reply:
I like it too!

Reply:
Thanks for the feedback!